apps/system/js/layout_manager.js

```diff
@@ -194,6 +203,16 @@
             this.publish('system-resize', systemResizeEventDetail);
           }
           break;
+
+        case 'mozfullscreenchange':
+          this.screen.classList.toggle('fullscreen-ancestor',
+            document.mozFullScreen);
+          if (evt.type === 'keyboardhide') {
```

Comment:
Whoops this is a mistake. I was just copying what happens in the `default` case to maintain current behavior for `'mozfullscreenchange':` (and I don't like not have a `break` and having case statements fall through). The publish for system-resize should be enough.